### DISCUSSION TITLE
External Scaler scaleAddress

### DISCUSSION THREAD

**Discussion by asipras** [kedacore/keda#4465] [General]
Hi is there an example of how to expose the scalerAddress on the Kubernetes Cluster?

I am using external scaler and it exposes grpc services on port 8080 for communication. However I do not know how to expose this service so that ScaledObject can reference it from the scalerAddress property.

Is there an example to do this?

Thanks.

**Comment by zroubalik** [comment]
You should use Kubernetes Service for this reference, if it is accesible from the namespace where your KEDA instance is running. If not, then maybe Ingress is needed.
